grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/ArmeriaMessageDeframer.java

```diff
@@ -325,6 +326,13 @@ public void close() {
         }
     }
 
+    /**
+     * Indicates whether or not this deframer is closing.
+     */
+    public boolean isClosing() {
```

Comment:
`isClosing()` in other API often implies that it will return `true` after it's closed. How about renaming it to something else? e.g. `needsToCloseWhenComplete()`?

Reply:
I think in that case we can just change it to return `true` even when closed. @ikhoon intentionally made the behavior this way but it isn't required https://github.com/line/armeria/pull/2685#discussion_r419298281 Don't think we need to make the name of the method longer

Reply:
I didn't know that `isClosing()` returns `true` in our API. 😅
Let me remove the line setting `closeWhenComplete` to `false`

Reply:
SGTM